src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -140,6 +140,7 @@ public static void Register(
                         prerelease.HasValue(),
                         highestPatch.HasValue(),
                         highestMinor.HasValue(),
+                        provider.LoadAuditSources().ToList(),
```

Comment:
Can you just make the args class take an IReadOnlyList?
The ToList is not necessary, since LoadAuditSources already returns a list.